default/python/AI/ResourcesAI.py

```diff
@@ -675,7 +675,8 @@ def set_planet_industry_and_research_foci(focus_manager, priority_ratio):
     target_pp = 0.001
     target_rp = 0.001
     resource_timer.start("Loop")  # loop
-    has_force = tech_is_complete("CON_FRC_ENRG_STRC")
+    # currently unused, but may be needed it again
+    # has_force = tech_is_complete("CON_FRC_ENRG_STRC")
```

Comment:
I don't like to have commented code. It looks very abandoned. 

We could just add docstring and something like that:
```
We could consider using CON_FRC_ENRG_STRC for more precise calculations.
```